gorapass/views.py

```diff
@@ -95,6 +95,17 @@ def user_completed_stamps(request, user_id):
 
     return HttpResponse('Unauthorized', status=401)
 
+def add_completed_hike(request, user_id):
```

Comment:
I'm a bit confused by the `user_id` parameter. Where does this value come from? Is this part of the magic of Django, that it can pull the user_id from the request without us explicitly setting it?

Reply:
No, this comes from the path defined in `urls.py`:

`path('users/<int:user_id>/completed_hikes/add', views.add_completed_hike, name='add_completed_hike')`

The `<int:user_id>` ends up passed in as a keyword argument, so the names have to match.